mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -118,12 +117,13 @@ case class ScalaPresentationCompiler(
       val driver = access.compiler()
       val uri = params.uri
       driver.run(uri, params.text)
-      implicit def ctx: Context = driver.currentCtx
 
+      implicit def ctx: Context = driver.currentCtx
```

Comment:
Same as above we can change these implicits here to `given` s.

Reply:
Ok, changed it too.